docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
```

Comment:
you know this version is deprecated? :)

Reply:
I didn't know https://docs.docker.com/compose/compose-file/compose-versioning/